framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -3329,6 +3366,44 @@ VkResult VulkanReplayConsumerBase::OverrideQueueSubmit2(PFN_vkQueueSubmit2 func,
         GetDeviceTable(queue_info->handle)->QueueWaitIdle(queue_info->handle);
     }
 
+    // Check whether any of the submitted command buffers are frame boundaries.
+    if (screenshot_handler_ != nullptr)
+    {
```

Comment:
This isn't the other place I saw a search of command buffers. I saw it in another file too.